File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] arr) {
        int n = arr.length;
        System.out.println(n);

        int sum = 0;
        for(int i = 0; i < arr.length; i++) {
            sum = sum  + arr[i];
        } 
        System.out.println(sum);

        int actualSum = (arr.length * (arr.length + 1))/2;
        System.out.println(actualSum);

        int missingNum = actualSum - sum;
        System.out.println(missingNum);

        return missingNum;
    }
}

File: 0268-missing-number/0268-missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(arr) {
    let n = arr.length;
    let sum = 0;

    for (let val of arr) sum += val;

    let actualSum = (n * (n + 1)) / 2;

    return actualSum - sum;
};